apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Logging/LabelProviders/UserLogEntryLabelProviderTest.cs

```diff
@@ -56,31 +56,6 @@ public void AddsUserAuthenticatedLabel()
             Assert.Equal(false.ToString(), label.Value);
         }
 
-        [Fact]
-        public void AddsUserNameLabelWithEmptyUserName()
-        {
-            // Arrange
-            var mockHttpContext = new Mock<HttpContext>();
-            mockHttpContext.Setup(x => x.User.Identity.IsAuthenticated).Returns(true);
-
-            var mockHttpContextAccessor = new Mock<IHttpContextAccessor>();
-            mockHttpContextAccessor.Setup(x => x.HttpContext).Returns(mockHttpContext.Object);
-
-            var instance = new UserLogEntryLabelProvider(mockHttpContextAccessor.Object);
-            var labels = new Dictionary<string, string>();
-
-            // Act
-            instance.Invoke(labels);
-
-            // Assert
-            var expected = new Dictionary<string, string>
-            {
-                { "user_authenticated", true.ToString() },
-                { "user_name", null }
```

Comment:
Instead of removing this test entirely, just expect the "user_authenticated" label.

Reply:
There is already a `AddsUserAuthenticatedLabel` test which tests that.

Reply:
You're right. I'll revert the delete and adjust the test.